Appendix/color articles (#71)
MrLongbottom/P10-Article: sections/appendix/color_articles.tex

\subsection{Coloring articles}\label{app:color_articles}
In this section, we are going to analyze a few more articles to get a more in depth view of how the topic distributions differ between the models.

> We are still looking at top three topics for each document and taking the 200 most probable words.

Reviewer: ```suggestion
We are still looking at the top three topics for each model and taking the 200 most probable words.
```

As a reminder the color table is shown below.
\begin{table}[h]
	\centering
	\caption{Color scheme for each model.}
	\begin{tabular}{l|c}
		Topic Model & Color \\
		\midrule
		\Acrlong{lda} & \thiscolor{Goldenrod} \vspace*{2mm} \\
		Author-Topic Model & \thiscolor{Aquamarine} \vspace*{2mm} \\
		Category-Topic Model & \thiscolor{LimeGreen} \vspace*{2mm} \\
		Word appearing in all models & \thiscolor{Peach} \vspace*{2mm}  \\
	\end{tabular}
	\label{tab:appendix_disc_color}
\end{table}
\noindent

> \autoref{art:1} shows an article about a race car driver from Aalborg, and him driving European Le Mans Series.

Reviewer: ```suggestion
\autoref{art:1} shows an article about a race car driver from Aalborg, and him driving in the European Le Mans Series.
```

> The article is not that long and therefore there are not many words marked.

Reviewer: ```suggestion
The article is not that long, and therefore there are not many words marked.
```

> From the colored words in the article, the words appearing in all models are not very descriptive of the what the articles is about.

Reviewer: ```suggestion
From the colored words in the article, the words appearing in all models are not very descriptive of what the article is about.
```

> Since the length of the article is short, we do not have many words co-occurring, which might indicate the marking of very few words.

Reviewer: ```suggestion
Since the length of the article is short, we do not have many words co-occurring, which might a reason why very few of these words are marked.
```

> The topic that is the most present in this article, is most likely a sports topic based on the top 10 words.

Reviewer: ```suggestion
The topic that is the most present in this article is most likely a sports topic based on the top 10 words.
```

> Top 10 words of the most present topic in the article: ['tour', 'vandt', 'løb', 'par', 'mig', 'løbet', 'hold', 'vm', 'slag', 'nummer'].

Reviewer: ```suggestion
The top 10 words of this most present topic are: ['tour', 'vandt', 'løb', 'par', 'mig', 'løbet', 'hold', 'vm', 'slag', 'nummer'].
```

> We can see based on these words in the top 10 words, other sports are higher up in the list which is probably because there are not many Le Mans articles.

Reviewer: ```suggestion
We can see based on these words that other sports are higher up in the list, such as 'tour' from Tour de France, which is probably because there are not many Le Mans articles.
```

\\
\begin{figure}
	\begin{tcolorbox}
		\emph{
			Dårligt \mycolor{Goldenrod}{LimeGreen}{løb} for Aalborg-kører SPIELBERG: Den aalborgensiske racerkører Anders Fjordbach havde sammen med teamet High Class Racing en gennemført skidt \mycolor{Goldenrod}{LimeGreen}{tredje} afdeling af European Le Mans Series på Red Bull Ring i Østrig. Her blev det til en beskeden ottendeplads i LMP2-klassen. - Det var \colorbox{Peach}{selvfølgelig} ærgerligt kun at blive \colorbox{Goldenrod}{nummer} otte, men det er \colorbox{LimeGreen}{vel} ikke en skam at have en dårlig weekend, siger Anders Fjordbach, der \colorbox{Goldenrod}{kører} sammen med Dennis Andersen.  Forkert dækvalg, et \colorbox{Peach}{mindre} sammenstød, en generator, der \colorbox{Aquamarine}{stod} af og andre små problemer var årsagen. Det \mycolor{Goldenrod}{Aquamarine}{eneste} positive var, at teamet \colorbox{Peach}{fortsat} er på tredjepladsen sammenlagt.clajen
		}
	\end{tcolorbox}

> 	\caption{Article 1}

Reviewer: ```suggestion
	\caption{Article 1.}
```

	\label{art:1}
\end{figure}

\autoref{art:2} is about politics in Aalborg, specifically about charter ships docking in Aalborg and how the municipality is trying to solve this problem.
\autoref{art:2} is a bit longer than the one in \autoref{art:1}, which in turn colors more words.

> We can see that the majority of the words are appearing in every model, where the words a lot of non descriptive words occur, such as "finde" (to find), "giver" (give) and "ting" (stuff). 

Reviewer: ```suggestion
We can see that the majority of these words are appearing in every model, where a lot of non-descriptive words occur, such as 'finde' (to find), 'giver' (give), and 'ting' (stuff). 
```

> Opposite to the original article analyzed in the paper, the combination of Category and \gls{lda} is much more present within this article.

Reviewer: ```suggestion
Opposite to the original article analyzed in the paper, word combinations of the Category-Topic model and \gls{lda} is much more present within this article.
```

> From the top words within the category model, many generic words appear, which might indicate the higher trend in words appearance.

Reviewer: ```suggestion
From the top words within the Category-Topic model, many generic words appear, which might be why there is a higher trend in word appearances.
```

> The author-topic model is not showing up with many unique words, but the author(Pernille K. Damsgaard) has written $922$ articles, which might indicate that she does not usually write about this topic.

Reviewer: ```suggestion
The Author-Topic model is not showing up with many unique words, but the author (Pernille K. Damsgaard) has written $922$ articles, which might indicate that she usually does not write about this topic.
```

		}
	\end{tcolorbox}

> 	\caption{Article 2}

Reviewer: ```suggestion
	\caption{Article 2.}
```

	\label{tab:appendix_disc_color}
\end{table}
\noindent
\autoref{art:1} shows an article about a race car driver from Aalborg, and him driving in the European Le Mans Series.
The article is not that long, and therefore there are not many words marked.
From the colored words in the article, the words appearing in all models are not very descriptive of what the article is about.

> Since the length of the article is short, we do not have many words co-occurring, which might a reason why very few of these words are marked.

Reviewer: ```suggestion
Since the length of the article is short, we do not have many words co-occurring, which might be a reason why very few of these words are marked.
```

		}
	\end{tcolorbox}
	\caption{Article 1.}
	\label{art:1}
\end{figure}

\autoref{art:2} is about politics in Aalborg, specifically about charter ships docking in Aalborg and how the municipality is trying to solve this problem.
\autoref{art:2} is a bit longer than the one in \autoref{art:1}, which in turn colors more words.

> We can see that the majority of these words are appearing in every model, where a lot of non-descriptive words occur, such as 'finde' (to find), 'giver' (give), and 'ting' (stuff). 

Reviewer: ```suggestion
We can see that the majority of these words appear in every model, where a lot of non-descriptive words occur, such as 'finde' (to find), 'giver' (give), and 'ting' (stuff). 
```

> Opposite to the original article analyzed in the paper, word combinations of the Category-Topic model and \gls{lda} is much more present within this article.

Reviewer: ```suggestion
Opposite to the original article analyzed in the paper, word combinations of the Category-Topic model and \gls{lda} model are much more present within this article.
```
